src/desktop/apps/auction_reaction/routes.tsx

```diff
@@ -0,0 +1,63 @@
+import { buildServerApp } from "reaction/Artsy/Router/server"
+import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
+import { routes } from "reaction/Apps/Auction/routes"
+import { stitch } from "@artsy/stitch"
+
+export const bidderRegistration = async (req, res, next) => {
+  if (!res.locals.sd.CURRENT_USER) {
```

Comment:
this is effectively a "route-level validation"? Don't allow a client to see the registration UI unless there's a logged in user? Would we extend this to render a modal when a user is already registered for the sale?